### PR TITLE
fix(flow): creating flow instance costs too much time

### LINKED ISSUE #979: [Bug]: creating a task ticket cost too much time
### ODC version

ODC 4.2.3

### OB version

-

### What happened?

image
It takes 18s to create a work order in team space.

### What did you expect to happen?

We want to speed up the creation of work orders.

### How can we reproduce it (as minimally and precisely as possible)?

Assign tasks in the team space.

### Anything else we need to know?

_No response_

### Cloud

_No response_

### PR DESCRIPTION
**Pull request by ungreat**
#### What type of PR is this?
Use batch creation to optimize the number of sql executions required to construct a topology

#### Which issue(s) this PR fixes:

Fixes #979 

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-common/src/main/java/com/oceanbase/odc/common/util/StringUtils.java [hunk 1]
```diff
@@ -336,4 +336,17 @@ public static Boolean checkPort(String port) {
         return matcher.matches();
     }
 
+    public static String removeWhitespace(String input) {
```
Comment (yizhouxw) [inline server/odc-common/src/main/java/com/oceanbase/odc/common/util/StringUtils.java:339]: unit testing missed

File: server/odc-service/src/main/java/com/oceanbase/odc/config/jpa/InsertSqlTemplateBuilder.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import javax.persistence.metamodel.SingularAttribute;
+
+import com.google.common.base.Joiner;
+import com.oceanbase.odc.common.util.StringUtils;
+
+public final class InsertSqlTemplateBuilder {
```
Comment (yizhouxw) [inline server/odc-service/src/main/java/com/oceanbase/odc/config/jpa/InsertSqlTemplateBuilder.java:29]: may move to `com.oceanbase.odc.common.jpa` package

Comment (yizhouxw) [reply server/odc-service/src/main/java/com/oceanbase/odc/config/jpa/InsertSqlTemplateBuilder.java:29]: also, please add unit testing cases

**Review by yizhouxw — COMMENTED** [review]
great job 👍

File: server/odc-common/src/main/java/com/oceanbase/odc/common/jpa/InsertSqlTemplateBuilder.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import java.util.stream.IntStream;
+
+import javax.persistence.metamodel.SingularAttribute;
+
+import com.google.common.base.Joiner;
+import com.oceanbase.odc.common.util.StringUtils;
+
+public final class InsertSqlTemplateBuilder {
+    private String tableName;
+    private List<String> filed = new ArrayList<>();
```
Comment (LuckyPickleZZ) [inline server/odc-common/src/main/java/com/oceanbase/odc/common/jpa/InsertSqlTemplateBuilder.java:31]: typo?

Comment (ungreat) [reply server/odc-common/src/main/java/com/oceanbase/odc/common/jpa/InsertSqlTemplateBuilder.java]: typo....

**Pull request closed without merging**